examples/resources/bloxone_keys_tsig/resource.tf

```diff
@@ -0,0 +1,11 @@
+resource "bloxone_keys_tsig" "example" {
+  name = "example_tsig_key"
+
+  # Other optional fields
+  comment   = "key created through terraform"
+  algorithm = "hmac_sha256"
+  secret    = "wuQuR0A08ApqKT65yaGiqWHalHxS7Ie8LF2VTUFZFZo="
```

Comment:
Can we avoid adding this field here? Maybe mention a comment that secret can be passed `or`
secret = "your secret value"